--- h4h5tools-1.1/config/commence.in.orig	2003-05-30 00:04:42.000000000 +0200
+++ h4h5tools-1.1/config/commence.in	2003-07-19 15:30:14.000000000 +0200
@@ -49,7 +49,7 @@
 LT_LINK_EXE=$(LT) --mode=link $(CC) $(LT_STATIC_EXEC) -dlopen self -rpath $(bindir)
 LT_RUN=$(LT) --mode=execute
 LT_INSTALL_PROG=$(LT) --mode=install $(INSTALL_PROGRAM)
-LT_INSTALL_LIB=$(LT) --mode=install $(INSTALL_DATA)
+LT_INSTALL_LIB=$(LT) --mode=install $(INSTALL_PROGRAM)
 LT_UNINSTALL=$(LT) --mode=uninstall $(RM)
 
 ## Optional variables. We must declare them here because Irix pmake
