--- h4h5tools-2.2.5/utils/h5toh4/h5toh4.c.orig	2020-03-10 15:33:18.000000000 +0100
+++ h4h5tools-2.2.5/utils/h5toh4/h5toh4.c	2024-12-18 06:17:54.922710383 +0100
@@ -302,7 +302,7 @@ main(int argc, char *argv[])
   {
     (void) fprintf(stderr, err1, argc);
     PrintOptions_h5toh4();
-    (void) fprintf(stderr, err4);
+    (void) fputs(err4, stderr);
     return EXIT_FAILURE;
   }
 
@@ -317,7 +317,7 @@ main(int argc, char *argv[])
     if ((token = gtoken(argv[i])) == TOK_ERR)
     {
       PrintOptions_h5toh4();
-      (void) fprintf(stderr, err4);
+      (void) fputs(err4, stderr);
       return EXIT_FAILURE;
     }
 
@@ -359,9 +359,9 @@ main(int argc, char *argv[])
 
       case TOK_ERR:   /* command syntax error */
       default:
-        (void) fprintf(stderr, err2);
+        (void) fputs(err2, stderr);
         PrintOptions_h5toh4();
-        (void) fprintf(stderr, err4);
+        (void) fputs(err4, stderr);
         return EXIT_FAILURE;
     }
   }
